projects/tuist/fixtures/app_with_spm_dependencies/Project.swift

```diff
@@ -21,6 +21,7 @@ let project = Project(
                 .external(name: "FirebaseDatabase"),
                 .external(name: "FirebaseFirestore"),
                 .external(name: "GoogleSignIn"),
+                .external(name: "Realm"),
```

Comment:
Can we try to also use `Realm` in the AppDelegate as we do for the other libraries to verify it links fine?